addons/payment/migrations/11.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<odoo>
+
+    <record id="payment_acquirer_payu" model="payment.acquirer">
+        <field name="description" type="html">
+            <p>
+                PayU India is an online payments solutions company serving the Indian market.
+            </p>
+            <ul class="list-inline">
+                <li><i class="fa fa-check"/>Online Payment
+                </li>
+                <li><i class="fa fa-check"/>Payment Status Tracking
+                </li>
+            </ul>
+        </field>
+        <field name="payment_icon_ids"
+               eval="[(6, 0, [ref(&quot;payment.payment_icon_cc_maestro&quot;),                                                         ref(&quot;payment.payment_icon_cc_mastercard&quot;),                                                         ref(&quot;payment.payment_icon_cc_american_express&quot;),                                                         ref(&quot;payment.payment_icon_cc_visa&quot;)])]"/>
+    </record>
+
+    <record id="payment_acquirer_custom" model="payment.acquirer">
+        <field name="description" type="html">
+            <p>
+                A generic payment acquirer to send payment instructions to any payment acquirer.
+            </p>
+            <ul class="list-inline">
+                <li><i class="fa fa-check"/>Online Payment
+                </li>
+            </ul>
+        </field>
+        <field name="auto_confirm"/>
```

Comment:
This is a false positive from the compare-noupdate-xml script. It notices that there is no value present anymore for this field, and as a result it adds an empty element. But the reason for the missing value is that the field was removed. In that case, it is better to remove these elements to prevent the following messages in the migration log

`WARNING openupgrade odoo.models: payment.acquirer.write() with unknown fields: auto_confirm`